scripts/themer

```diff
@@ -84,7 +84,7 @@ def get_filters():
         return get_component(color, 2)
 
     def to_float(num):
-        return num / 255
+        return num / 255.
```

Comment:
@Amar1729 oh, you are still using python 2. In python 3 float is the default and you need to use `5 // 2` to force integer division.

Instead of doing this, `division` should be added to the `from __future__ import` above.